crunch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Is this actually required?

Reply:
This forces C++14 standard. You probably don't want that :)

Reply:
@boxerab Ok thank you for the prompt answer, I'll remove it then :-)